src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -513,13 +510,23 @@ def to_white_list(
             for idx, host in enumerate(hosts if hosts else default_hosts)
         ]
 
+    def to_ingress(self) -> V1AlphaIngressRoute | None:
```

Comment:
```suggestion
    def to_ingress(self) -> Optional[V1AlphaIngressRoute]:
```

Reply:
So I actually had this, but then I changed it. For return types it feels more natural to me to state it returns `V1AlphaIngressRoute` or nothing instead of it optionally returns `V1AlphaIngressRoute`. `Optional[V1AlphaIngressRoute]` feels better for inputs. But maybe I'm weird 😬

Reply:
I checked the documentation and there's no mention of a preferred or "pythonic" way of doing it, so it's pretty much a personal preference. In which case do ignore me and keep this 👍